example/cat.sw

```diff
@@ -11,7 +11,7 @@ let randdir : cmd dir = {
 
 forever {
   n <- random 20;
-  repeat (10 + n) wait;
+  wait (10 + n);
```

Comment:
Good catch.  We should probably add something to the CI so it runs some kind of check on every file in `example/` (but we can do that separately).

Reply:
Created #180